stock_return_request/models/stock_move.py

```diff
@@ -16,7 +16,10 @@ class StockMove(models.Model):
     def _compute_qty_returnable(self):
         """Looks for chained returned moves to compute how much quantity
         from the original can be returned"""
-        for move in self.filtered(lambda x: x.state not in ["draft", "cancel"]):
```

Comment:
@sergio-teruel 
To avoid multiple `__set__` calls and n condition evaluations, do this:

`self.update({"qty_returnable": 0.0})`

Reply:
AFAIK, doing `self.qty_returnable = 0` has the same performance against `update`, as there's only one value to set.